app/assets/javascripts/battletype/combat_zone.js

```diff
@@ -3,7 +3,10 @@
     _properties: {
       randomFreeVerticalSlot: {
         get: function () {
-          return Math.floor(Math.random() * this.height()) - 80;
+          var hudHeight = 60;
```

Comment:
Why do we use hard-coded values for a part of the UI (`hudHeight`), but dynamically evaluated values for the rest of it (`this.height()`)?

Since the UI is pretty stable anyway, I'd go with hard-coded values (assigned to explicit constants) everywhere, but it's just my personal taste; I'd be fine with using dynamic values everywhere, too, as long as we're consistent.